/**
 * This file is part of the "sensord" project
 *   Copyright (c) 2015 Finn Zirngibl
 *   Copyright (c) 2015 Paul Asmuth
 *
 * FnordMetric is free software: you can redistribute it and/or modify it under
 * the terms of the GNU General Public License v3.0. You should have received a
 * copy of the GNU General Public License along with this program. If not, see
 * <http://www.gnu.org/licenses/>.
 */
package sensord;

message SampleRule {

  /**
   * The sensor that should be sampled
   */
  required string sensor_key = 1;

  /**
   * The sampling interval for this rule in milliseconds
   */
  required uint64 interval = 2;

}

message SamplerConfig {
  repeated SampleRule rules = 1;
}

